js/modal.js

```diff
@@ -0,0 +1,538 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] Sometime linters same to be grating :smile: But it's for better cause, it help you write more readable code and follow the standard style. Therefore it would be good to find solution to errors pointed out by linters than disable them.

Reply:
Thanks for the reminder, but that was an auto fix text.  As you know, windows always change LF to CRLF.